### PR TITLE
docs: remove direct calls to require

### PR DESCRIPTION
**Pull request by felipenoris**
Regarding the changes on _workflow-tips_:
- `isinteractive()` evaluates to false
- a call to `include("file.jl")` on `.juliarc.jl` looks for files at the user's home folder, so I added a call to `pwd()`.

### CODE REVIEW

**Comment by tkelman** [comment]
> `isinteractive()` evaluates to false

When? It's supposed to be true for a REPL session and false for scripts.

**Comment by felipenoris** [comment]
Inside `.juliarc.jl`:

```julia
~$ cat .juliarc.jl

if isinteractive()
	println("Interactive")
else
	println("Not interactive")
end
```

```julia
~$ ~/Documents/src/julia/julia
Not interactive
```

**Comment by tkelman** [comment]
Ah. Wonder if/when that changed?

**Comment by yuyichao** [comment]
https://github.com/JuliaLang/julia/issues/10779

**Comment by felipenoris** [comment]
Looks like a regression. I should put it back in the docs.

**Comment by yuyichao** [comment]
It's intentional afaict.

**Comment by felipenoris** [comment]
So I guess there's nothing to change for this PR.

**Comment by yuyichao** [comment]
If you only want to use it in repo, you should call it in `atreplinit` intead.

File: doc/manual/workflow-tips.rst [hunk 1]
```diff
@@ -75,7 +75,7 @@ issuing the command::
 
 If you further add the following to your ``.juliarc.jl`` file ::
 
-    isinteractive() && isfile("_init.jl") && require("_init.jl")
+    isfile("_init.jl") && include(joinpath(pwd(), "_init.jl"))
```
Comment (tkelman) [inline doc/manual/workflow-tips.rst]: I'd just do `include("_init.jl")`, why is the `joinpath` needed?

Comment (yuyichao) [inline doc/manual/workflow-tips.rst]: include is relative to current file. so at least this makes it consistent with the `isfile` check. I'm not sure which one (relative to current file or PWD) is the goal here.

Comment (felipenoris) [inline doc/manual/workflow-tips.rst]: The example on the docs puts the case when `_init.jl` is in the current dir:
`...then calling julia from that directory...`
I figured `isfile` checks current dir, but `include` will check the dir where `.juliarc.jl` is located.

**Pull request merged**